powerhome/playbook
# Nav Kit Update

#### Screens

<img width="1551" alt="Screen Shot 2020-10-13 at 10 42 04 PM" src="https://user-images.githubusercontent.com/53874143/95937452-d937a680-0da5-11eb-8958-f9ea052cbd8b.png">
<img width="1552" alt="Screen Shot 2020-10-13 at 10 43 05 PM" src="https://user-images.githubusercontent.com/53874143/95937472-e05eb480-0da5-11eb-95b3-6beba13bbbee.png">

#### Breaking Changes

No, styles that were changed (ie padding and text style) were only visual. Other additions are more options added to the kit.

#### Runway Ticket URL

[NUXE-161](https://nitro.powerhrg.com/runway/backlog_items/NUXE-161)

#### How to test this

Borderless: Visually checked the borderless options to make sure no lines were present.
Custom Icon: Inserted image url's to ensure the image is being passed and it was the correct size.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
